src/gluelayer.d

```diff
@@ -55,14 +55,11 @@ version (NoBackend)
 }
 else
 {
-    import ddmd.backend.type;
+    public import ddmd.backend.cc : block, Blockx, Symbol;
```

Comment:
Specifically, don't we have some bugs with selective imports that affect some of the older host compilers we still support?  Or am I misremembering?